### ISSUE TITLE
curriculum training

### ISSUE THREAD

**Issue by Jiakui** [Canjie-Luo/MORAN_v2#5]
In the paper, it is said that the model should be trained with three stages. However, in the code, I didn't see how you implement the three stages. Therefore, if we run the code, could we reproduce the accuracy mentioned in the paper?

Thanks!

**Comment by Canjie-Luo** [comment]
Yes, the MORAN v1 in our paper was trained using curriculum learning. But the review took a long time. And we developed and released MORAN v2 for more stable convergence and higher accuracy. The released version was trained end-to-end with only one stage. But it outperformed the MORAN v1. The results were reported in the `README.md`. Run the code and manually decrease the learning rate~You can reproduce the v2 results. By the way, if the MORAN v2 is trained using curriculum learning, it may yield better performance.

**Comment by BlakeXiaochu** [comment]
Hi~ We can reproduce the v2 results by runnuing the code and manually decreasing the learning rate. What's the strategy of decreasing learning rate? For example, divide the learning rate by 10 every 100000 iter? Thx~

**Comment by Canjie-Luo** [comment]
Please divide the lr by 10 after 3 epoch. Totally 6 epoch maybe enough for training. Enjoy yourself~
